### ISSUE TITLE
Data folder not included in .egg

### ISSUE THREAD

**Issue by JonasKemmer** [mlafarga/raccoon#2]
Hey,

thanks for providing your CCF code on github!
I just run into trouble when I tried to run your example. Using only the Mask ID returned an OSError (not found) because the "data" folder was not copied into the compiled package in Anaconda.

 `OSError: .../anaconda3/envs/raccoon/lib/python3.6/site-packages/raccoon-0.0.1-py3.6-linux-x86_64.egg/raccoon/data/mask/CARM_VIS/J12123+544Sserval_tellbervmax_fwhm2.00-30.00_contrminmin0.06_depthwq0.60_contrastmeanfwhm-1.mas not found.`

I copied it from the source and now it works.

Best,
Jonas

**Comment by rluquer** [comment]
Yes, I confirm this. Copying from the source makes the code work.

**Comment by mlafarga** [comment]
Should have been solved with the new meson installation/build.
